integration-web/src/main/java/org/symphonyoss/integration/web/resource/WebHookResource.java

```diff
@@ -290,4 +290,19 @@ public void run() {
     LOGGER.error(ex.getMessage(), ex);
     return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("Unexpected exception");
   }
+
+
+  /**
+   * Handle {@link IntegrationUnavailableException} exception.
+   * @param ex Exception object
+   * @return HTTP 503 (Service Unavailable)
+   */
+  @ResponseBody
+  @ExceptionHandler(IntegrationUnavailableException.class)
+  public ResponseEntity<String> handleUnavailableException(IntegrationUnavailableException ex) {
```

Comment:
You should consider adding a new exception on 'handleIntegrationBridgeUnavailableException' method.

Btw, you should rename this method to 'handleServiceUnavailableException'.